nursery/persist-via-default-file-association-registry-key.yml

```diff
@@ -0,0 +1,20 @@
+rule:
+  meta:
+    name: persist via default file association registry key
+    namespace: persistence/registry
+    authors:
+      - j.j.vannielen@utwente.nl
+    scopes:
+      static: function
+      dynamic: call
+    att&ck:
+      - Persistence::Event Triggered Execution::Change Default File Association [T1546.001]
+    references:
+      - https://woshub.com/managing-default-file-associations-in-windows-10/
```

Comment:
is this the right reference? it seems to talk about other registry paths/options